pkgs/applications/video/plex-media-player/default.nix

```diff
@@ -72,7 +62,6 @@ in stdenv.mkDerivation rec {
   postInstall = ''
     mkdir -p $out/share/{applications,pixmaps}
```

Comment:
Then you don't need to create the `application` directory anymore.
You can also combine the remaining two lines with

```
install -D ${src}/resources/images/icon.png $out/share/pixmaps/plex-media-player.png
```

Reply:
Thanks!